teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+# Represents a teacher, inherited from the Person class.
+class Teacher < Person
+  def initialize(age, specialization, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
```

Comment:
- Not sure why you are creating class "teacher". you are supposed to have these classes: `Classroom`, `Student`, `Person`, `Book` and `Rental`. Kindly check again the project [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md) and implements all required classes and methods..

Reply:
This was in the previous instructions not In this PR, its not meant for this PR